cms/api.py

```diff
@@ -28,8 +28,9 @@ def get_course_enrollment_text(course):
                 text += ' - Enrollment {:%m/%Y}'.format(
                     course_run.enrollment_start
                 )
-        elif course_run.fuzzy_start_date:
-            text = 'Coming ' + course_run.fuzzy_start_date
+    else:
+        course_run = course.get_promised_run()
```

Comment:
This is not a guarantee, we need to handle the case where the return value is `None` here